src/hssm/hssm.py

```diff
@@ -516,6 +517,37 @@ def sample_posterior_predictive(
             idata = self._inference_obj
         return self.model.predict(idata, kind, data, inplace, include_group_specific)
 
+    def sample_prior_predictive(
+        self,
+        draws: int = 500,
+        var_names: str | list[str] | None = None,
+        omit_offsets: bool = True,
+        random_seed: np.random.Generator | None = None,
+    ) -> az.InferenceData:
+        """Generate samples from the prior predictive distribution.
+
+        Parameters
+        ----------
+        draws
+            Number of draws to sample from the prior predictive distribution. Defaults
+            to 500.
+        var_names
+            A list of names of variables for which to compute the prior predictive
+            distribution. Defaults to ``None`` which means both observed and unobserved
+            RVs.
+        omit_offsets
+            Whether to omit offset terms in the plot. Defaults to ``True``.
```

Comment:
plot?

Reply:
This is directly copied from bambi documentation lol. I don't know what this refers to

Reply:
prob just take 'plot' out? 
Can add back in if it turns out to make sense :)

Reply:
Will do